valuable/src/value.rs

```diff
@@ -65,6 +65,27 @@ macro_rules! value {
                 $variant($ty),
             )*
 
+            /// An arbitrary value that implements Display.
```

Comment:
nit: this should probably be
```suggestion
            /// An arbitrary value that implements [`std::fmt::Display`].
```